### PR TITLE
Fixes in python benchmark

### PR DESCRIPTION
**Pull request by eddyxu**

### CODE REVIEW

File: python/benchmarks/oxford_pet.py [hunk 1]
```diff
@@ -6,17 +6,29 @@
...
 
+import lance
+
 oxford_pet_benchmarks = BenchmarkSuite("oxford_pet")
 
 
+@oxford_pet_benchmarks.benchmark("total_count", key=['fmt', 'flavor'])
+def total_count(base_uri: str, fmt: str, flavor: Optional[str]):
+    if fmt == "raw":
+        return get_pets_class_distribution(base_uri)
```
Comment (changhiskhan) [inline python/benchmarks/oxford_pet.py:23]: This doesn't count number of rows ?

**Review by changhiskhan — COMMENTED** [review]
Small nit

File: python/benchmarks/oxford_pet.py (hunk 1 above)
Comment (eddyxu) [reply python/benchmarks/oxford_pet.py:23]: Oh, i copied this piece of code from class distribution, and have not add raw performance number.

Will add it later.

**Comment by eddyxu** [comment]
Has been addressed in other PRs.

**Pull request closed without merging**